### ISSUE TITLE
"npm install webrtc-native --save" doesn't work

### ISSUE THREAD

**Issue by kinleyradams** [vmolsa/webrtc-native#51]
I get `  throw new Error('prebuilt module not found. See the instructions from https://github.com/vmolsa/webrtc-native/wiki/Getting-started#building-from-source for building module from source.');
      ^
Error: prebuilt module not found. See the instructions from https://github.com/vmolsa/webrtc-native/wiki/Getting-started#building-from-source for building module from source.
`
when i try to run npm install webrtc-native --save

**Comment by timur-han** [comment]
I receive the same error, it seems like the URL http://cide.cc:8080/webrtc/ is not available...

**Comment by ghost** [comment]
@timur-han Did you end up finding a workaround?

**Comment by timur-han** [comment]
@swangyal nope, the project seems to be inactive and not maintained anymore

**Comment by caoyongfeng0214** [comment]
me too

**Comment by nunahsan** [comment]
any update on this guys ?
